internal/controller/constant/odlm.go

```diff
@@ -1768,11 +1768,12 @@ spec:
                   required: true
                   configMapKeyRef:
                     name: cloud-native-postgresql-image-list
-                    key: ibm-postgresql-14-operand-image
+                    key: ibm-postgresql-16-operand-image
                     namespace: {{ .OperatorNs }}
                 configMapKeyRef:
-                    name: ibm-cpp-config
-                    key: edb-keycloak-operand-image
+                  name: cloud-native-postgresql-operand-images-config
+                  key: ibm-postgresql-16-operand-image
+                  namespace: {{ .OperatorNs }}
```

Comment:
Can we keep using Postgres 14 image for Keycloak EDB Cluster CR?

Direct upgrade between Postgres major versions is not supported.